snuba/web/rpc/storage_routing/routing_strategies/storage_routing.py

```diff
@@ -227,6 +235,28 @@ def _is_highest_accuracy_mode(self, in_msg_meta: ResponseMeta) -> bool:
             == DownsampledStorageConfig.MODE_HIGHEST_ACCURACY
         )
 
+    def get_allocation_policies(cls) -> list[AllocationPolicy]:
```

Comment:
I see that this is an instance method but the parameter is named as `cls`, which is usually used for class methods. Rename `cls` to `self`. Or if this must be a class method, then add `@classmethod `decorator.